examples/docker-build/components/Logo.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
To keep example small, I suggest removing extra boilerplate like this from the example.